### PR TITLE
Dtserwfour 170 remaining screen updates

### PR DESCRIPTION
**Pull request by pdamodarannair**

### CODE REVIEW

**Review by vwagh-hw — DISMISSED** [review]
LGTM but please verify that all Automation tests pass as well

**Review by shanavas123 — APPROVED** [review]

**Review by rajkumar-balakrishnasamy — DISMISSED** [review]

**Review by vwagh-hw — DISMISSED** [review]

**Review by shanavas123 — DISMISSED** [review]

**Review by shanavas123 — APPROVED** [review]

**Review by rajkumar-balakrishnasamy — APPROVED** [review]

**Review by shanavas123 — DISMISSED** [review]

**Review by shanavas123 — DISMISSED** [review]

**Review by shanavas123 — DISMISSED** [review]

File: commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java [hunk 1]
```diff
@@ -90,12 +90,27 @@ public void testAllCurrencyFormats() {
         }
     }
 
+    @Test
+    public void testAllCurrencyFormatsWithSymbol() {
+        String amount = "1000000";
+            Context context = ApplicationProvider.getApplicationContext();
+            String currency = CurrencyParser.getInstance(context).formatCurrencyWithSymbol("TWD", amount);
+            assertThat(currency, is("NT$1,000,000"));
+    }
```
Comment (vwagh-hw) [inline commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java:99]: method names says test all currency formats but only 1 assert. Rename method name
Formatting looks wrong.

File: commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java [hunk 2]
```diff
@@ -90,12 +90,27 @@ public void testAllCurrencyFormats() {
...
     public void testGetNumberOfFractionDigits_ThreeDigitDecimal(){
         Context context = ApplicationProvider.getApplicationContext();
         int noOfDigits = CurrencyParser.getInstance(context).getNumberOfFractionDigits("TND");
         assertThat(noOfDigits, is(3));
     }
+
+    @Test
+    public void testGetCurrency() {
+        Context context = ApplicationProvider.getApplicationContext();
+        CurrencyDetails currency = CurrencyParser.getInstance(context).getCurrency("TND");
```
Comment (vwagh-hw) [inline commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java:110]: var name should bee `currencyDetails`

Comment (pdamodarannair) [reply commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java:110]: It is not variable, it is a  class name.

File: commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java [hunk 3]
```diff
@@ -105,10 +105,11 @@ public void testGetNumberOfFractionDigits_ThreeDigitDecimal(){
     }
 
     @Test
-    public void testGetCurrency() {
+    public void testCurrencyDetails() {
```
Comment (vwagh-hw) [inline commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java:108]: we test methods in unit tests and so this should remain `testGetCurrency`

File: commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java (hunk 2 above)
Comment (vwagh-hw) [reply commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java:110]: yes I know, I am asking you to rename the variable 
`CurrencyDetails currency =`
Change to 
`CurrencyDetails currencyDetails =`

File: commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java (hunk 3 above)
Comment (pdamodarannair) [reply commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java]: Resolved

File: commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java (hunk 2 above)
Comment (pdamodarannair) [reply commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java]: Done

**Review by vwagh-hw — APPROVED** [review]

**Review by malaw-hw — APPROVED** [review]

**Pull request merged**